tardis/conftest.py

```diff
@@ -190,9 +190,17 @@ def tardis_config_verysimple():
 
 
 @pytest.fixture(scope="function")
```

Comment:
change scope to "module"

Reply:
This should be done in a separate PR since this occurs in other, similar functions as well that are unrelated to this PR. I've opened an issue on this